packages/server-sdk/src/lib/public.types.ts

```diff
@@ -88,6 +88,15 @@ export interface AuthToken {
     tokenType: string
 }
 
+export interface ProviderCallback {
```

Comment:
for consistency can we change here as well since these are not callbacks

Reply:
Done